ext/hal/microchip/mec/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+
+zephyr_include_directories_ifdef(CONFIG_SOC_SERIES_MEC1701X ./)
```

Comment:
Seems like this might belong in the CMakeLists a dir up?

Reply:
Hi,
I did it this way because MEC1701 is using an old, single header and we are focusing on MEC1501. 
If I move the MEC1701 up a directory would it only include the directory containing MCHP_MEC1701.h?  For example in ext/hal/microchip/ add zephyr_include_directories_ifdef(CONFIG_SOC_SERIES_MEC1701X mec) ?